Packs/NucleonCyber/Integrations/NucleonCyberFeed/NucleonCyberFeed.py

```diff
@@ -0,0 +1,761 @@
+https://xsoar.pan.dev/docs/integrations/code-conventions#outputs
+https://xsoar.pan.dev/docs/integrations/dt
+
+
+Main Function
+-------------
+
+The ``main()`` function takes care of reading the feed integration parameters via
+the ``demisto.params()`` function, initializes the Client class and checks the
+different options provided to ``demisto.commands()``, to invoke the correct
+command function passing to it ``demisto.args()`` and returning the data to
+``return_results()``. ``main()`` also catches exceptions and
+returns an error message via ``return_error()``.
+
+
+Entry Point
+-----------
+
+This is the integration code entry point. It checks whether the ``__name__``
+variable is ``__main__`` , ``__builtin__`` (for Python 2) or ``builtins`` (for
+Python 3) and then calls the ``main()`` function. Just keep this convention.
+
+"""
+
+from typing import Dict, List, Optional
+# from datetime import date
+import urllib3
+import json
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+# disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any Demisto logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this HelloWorld Feed implementation, no special attributes defined
+    """
+
+    def build_iterator(self) -> List:
+        """Retrieves all entries from the feed.
+        Returns:
+            A list of objects, containing the indicators.
+        """
+
+        result = []
+
+        res = self._http_request('GET',
+                                 url_suffix='',
+                                 full_url=self._base_url,
+                                 resp_type='text',
+                                 )
+
+        # In this case the feed output is in text format, so extracting the indicators from the response requires
+        # iterating over it's lines solely. Other feeds could be in other kinds of formats (CSV, MISP, etc.), or might
+        # require additional processing as well.
+        try:
+            indicators = res.split('\n')
+
+            for indicator in indicators:
+                # Infer the type of the indicator using 'auto_detect_indicator_type(indicator)' function
+                # (defined in CommonServerPython).
+                if auto_detect_indicator_type(indicator):
+                    result.append({
+                        'value': indicator,
+                        'type': auto_detect_indicator_type(indicator),
+                        'FeedURL': self._base_url
+                    })
+
+        except ValueError as err:
+            demisto.debug(str(err))
+            raise ValueError(f'Could not parse returned data as indicator. \n\nError massage: {err}')
```

Comment:
add from err for more enriched stack trace for debugging purposes if needed later on
```suggestion
            raise ValueError(f'Could not parse returned data as indicator. \n\nError massage: {err}') from err
```